keps/sig-api-machinery/4006-transition-spdy-to-websockets/README.md

```diff
@@ -916,8 +1068,8 @@ Pick one more of these and delete the rest.
 -->
 
 - [X] Metrics
-  - Metric name: TBD (complete for beta)
-  - Components exposing the metric: kube-apiserver
+  - Metric name: None
+  - Components exposing the metric: N/A
```

Comment:
Let's add a metric. I'll comment on the below list of metrics.

Reply:
Added four metrics: two for successful connection upgrades and two for connection failures. Please let me know what you think.